<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_FullAuto</class>
 <widget class="QDialog" name="Dialog_FullAuto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>全自动三维重建</string>
  </property>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>（极高品质可能会花费非常长的时间）</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>80</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>(可忽略)</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_describer">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>SIFT</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SIFT_ANATOMY</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AKAZE_FLOAT</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AKAZE_MLDB</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_geometricmodel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>fundamental matrix</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>esential matrix</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>homography matrix</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>essential matrix with an angular parametrization</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>orthographic essential matrix</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>upright essential matrix</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit_inputDir">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>50</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_browseOutputDir">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>130</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>特征点质量:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>特征暴力计算:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>100</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>输入格式: f;0;ppx;0;f;ppy;0;0;1</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_forcecompute">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>否</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>是</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_upright">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <item>
    <property name="text">
     <string>否</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>是</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit_distanceRatio">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0.8</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_forcematch">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>270</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>否</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>是</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>距离比率:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>130</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>工程输出路径:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>计算特征点参数设置:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>计算特征空间向量:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_browseInputDir">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>50</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>20</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>工程路径参数设置：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>（开启后，若找不到特征，则强制取距离最近的点）</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_quality">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>默认</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>高品质</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>极高品质</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>50</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>源图片路径:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>匹配特征点参数设置:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>240</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>（舍弃无意义匹配点的距离比率，使用0.6或更低比率可以减少误报）</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>80</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>相机本征矩阵:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_kMatrix">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>80</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>几何模型:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>暴力匹配:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_OutputDir">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>130</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>（旋转的特征标志）</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>特征点算子:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>（开启后，匹配不到特征会强制重算数据）</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>空中三角测量方法:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>计算后方交会方法:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>三维重建参数设置:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_triangulationMethod">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>340</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <item>
    <property name="text">
     <string>DIRECT_LINEAR_TRANSFORM</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>L1_ANGULAR</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LINFINITY_ANGULAR</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>INVERSE_DEPTH_WEIGHTED_MIDPOINT</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_resectionMethod">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>370</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <item>
    <property name="text">
     <string>DLT_6POINTS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>P3P_KE_CVPR17</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>P3P_KNEIP_CVPR11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>P3P_NORDBERG_ECCV18</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>UP2P_KUKELOVA_ACCV10</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btn_CANCEL">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>330</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_OK">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>330</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>确定并开始工程</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>420</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>ply</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>obj</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>osgb</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>纹理模型输出格式:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
